tests/unit/storage/data/locConstraintParser.spec.js

```diff
@@ -38,22 +38,22 @@ describe('locationConstraintParser', () => {
         const client = clients[awsLocation];
         assert.notStrictEqual(client, undefined);
         assert(client instanceof AwsClient);
-        assert.strictEqual(client._s3Params.sslEnabled, true);
-        assert.strictEqual(client._s3Params.httpOptions.agent.protocol,
-            'https:');
-        assert.strictEqual(client._s3Params.httpOptions.agent.keepAlive, false);
-        assert.strictEqual(client._s3Params.signatureVersion, 'v4');
+        assert(client._s3Params.endpoint.startsWith('https://'));
+        if (client._s3Params.httpOptions && client._s3Params.httpOptions.agent) {
```

Comment:
should not have a condition: either this is a HTTP test, or it is an HTTPS one...